api_tester/lib/api_tests/purchases_flutter_api_test.dart

```diff
@@ -144,6 +144,11 @@ class _PurchasesFlutterApiTest {
     future = Purchases.setAutomaticAppleSearchAdsAttributionCollection(enabled);
   }
 
+  void _checkEnableAdServicesAttributionTokenCollection() {
+    Future<void> future;
+    future = Purchases.enableAdServicesAttributionTokenCollection();
```

Comment:
Future<void> future = Purchases.enableAdServicesAttributionTokenCollection();

Reply:
I think we want to specify the type of the future since this is part of the API tests. We can maybe make it in a single line... Not sure why it was separated in other methods